test/integration/stdio_test.exs

```diff
@@ -0,0 +1,83 @@
+defmodule Membrane.File.Integration.StdioTest do
+  use ExUnit.Case, async: false
+  require Logger
+
+  # 5min if dev env needs to compile
+  @moduletag timeout: 360_000
+
+  @input_text_file "test/fixtures/input.txt"
+  @cmd_out_prefix "/tmp/membrane_file_test_out"
+  @cmd_err_prefix "/tmp/membrane_file_test_err"
+
+  setup_all(_context) do
+    assert Mix.Task.run("compile", ["test/fixtures/*.exs"]) in [:ok, :noop]
+    :ok
+  end
+
+  setup(%{line: line} = _context) do
```

Comment:
You can use `@tmp_dir` tag and obtain some temporary directory instead of generating the file names